exercises/rect/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "fmt"
+
+func Rect(x, y int) [][]int {
```

Comment:
If `x` or `y` are negative, your program will crash.

Please make sure that your program handle negative inputs correctly (the function may return an empty slice in this case).